app/views/settings/_avatar.slim

```diff
@@ -1,10 +1,15 @@
 .card.h-100
   .card-header: h4.my-0.font-weight-normal Avatar
   .card-body
-    ' Your avatar is linked to your
     - if current_user.twitch
-      ' Twitch
+      ' Your avatar is linked to your Twitch account and can be updated
+      ' there. After any changes, you may need to log out of Splits.io and then
+      ' back in to refresh it.
+    - elsif current_user.google
+      ' Your avatar is linked to your Google account and can be updated
+      ' there. After any changes, you may need to log out of Splits.io and then
+      ' back in to refresh it.
     - else
-      ' Google
-    ' account and can be updated there. After any changes, you may need to log out of Splits.io and then back in to
-    ' refresh it.
+      / TODO: Add Gravatar support
```

Comment:
Have you tested out how the rest of the site will handle null avatars?

Reply:
I'm fairly confident Twitch can (used to?) report null avatars already, so I believe the code already knows how to handle it. But yeah I've done a little manual testing on this branch to be sure, it displays a fallback.

Reply:
Sounds good, just wanted to make sure random pages wouldn't start throwing 500's.